mael/recaptcha-bundle/0.1/post-install.txt

```diff
@@ -0,0 +1,3 @@
+<bg=blue;fg=white> Next for MaelRecaptchaBundle </>
+    * <fg=blue>Edit mael_recaptcha.yaml</> add you keys
```

Comment:
Should keys be edited in the `.env` file?

Reply:
Oh yes, I forgotten. I'am going to make the modification now